README.md

```diff
@@ -345,12 +345,7 @@ clasp <command>
 
 #### Run Tests (experimental)
```

Comment:
Can we graduate the tests from experimental?

Reply:
👍